ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityContent.component.spec.ts

```diff
@@ -3,38 +3,58 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { ActivityContentBlock } from '../../../models/activity/activityContentBlock';
 import { ActivityContentComponent } from './activityContent.component';
+import {FileDownloadService} from '../../../services/fileDownload.service';
+import {of} from 'rxjs';
+import {ActivitySection} from 'ddp-sdk';
+import {DownloadFileComponent} from './downloadFile.component';
+import {expect} from '@angular/flex-layout/_private-utils/testing';
 
 describe('ActivityContentComponent', () => {
     const contentBlock = {
         content: 'Some text',
         title: null
     } as ActivityContentBlock;
 
+    const section = {
+        blocks: []
+    } as ActivitySection;
+
     @Component({
         template: `
-        <ddp-activity-content [block]="block">
+        <ddp-activity-content  [studyGuid]="'CMI-LMS'" [block]="block">
```

Comment:
Please add two additional tests:
1) for CMI-OSTEO that covers the same check that the download link is visible
2) for a study we don't want the download link to be visible and confirm it isn't there. (e.g. a negative check)

Reply:
This `studyGuid` doesn't do anything, as we don't check that in the component. Even if I wrote in this test instead of 'CMI-LMS' something like 'BLA-BLA-BLA', the test still will pass. 

**This is the most important check on the component level:**
```
this.activityCode === 'SOMATIC_RESULTS' && this.activityStatusCode === 'COMPLETE' &&
            !!showResultsBlock && showResultsBlock.stableId === 'SHOW_RESULTS' && showResultsBlock.answer === true;
```

Reply:
Fair enough.  I thought you had added a control based on the studyGuid but reviewing the code again makes me realize that's not the case.